src/viz/layer/Layer.ts

```diff
@@ -29,7 +29,7 @@ import { basicStyle } from '../style/helpers/basic-style';
 export enum LayerEvent {
   DATA_READY = 'dataReady',
   DATA_CHANGED = 'dataChanged',
-  TILES_LOADED = 'tilesLoaded',
+  // TILES_LOADED = 'tilesLoaded',
```

Comment:
So eventually no use at all of this...? 

I guess that if we need to 'refresh' the tiles without changing the viewport (for example when requiring new fields in the map), we are rising an equivalent data changed event right?

If that's it, we should completely remove TILES_LOADED event

Reply:
ok, I've removed the comments